src/Saas.Admin/Saas.Admin.Service/Services/IPermissionService.cs

```diff
@@ -2,9 +2,9 @@
 
 public interface IPermissionService
 {
-    Task<IEnumerable<string>> GetTenantUsersAsync(Guid tenantId);
-    Task<IEnumerable<string>> GetUserPermissionsForTenantAsync(Guid tenantId, string userId);
-    Task AddUserPermissionsToTenantAsyc(Guid tenantId, string userId, string[] permissions);
-    Task RemoveUserPermissionsFromTenantAsync(Guid tenantId, string userId, string[] permissions);
-    Task<IEnumerable<Guid>> GetTenantsForUserAsync(string userId, string? filter);
+    Task<IEnumerable<string>> GetTenantUsersAsync(string tenantId);
```

Comment:
they should really be guids on the interface, right?

Reply:
See comment above